tests/publishdependency.targets

```diff
@@ -9,23 +9,13 @@
   </ItemGroup>
 
   <ItemGroup>
-    <AllProjectJsonLockFiles Include="$(SourceDir)**\*\project.lock.json"/>
+    <ProjectLockJsonFiles Include="$(SourceDir)\runtime\project.lock.json"/>
```

Comment:
To confirm, this is being used to restore dependency when creating the overlay, correct?